### PR TITLE
Add citations to documentation

### PR DESCRIPTION
**Pull request by FBartos**

### CODE REVIEW

File: inst/help/summarystatsgeneralbayesiantests.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Summary Statistics General Bayesian Tests
 ====================================
 
-The general Bayesian tests allow one to test a hypothesis about a parameter under different data likelihoods.
+The general Bayesian tests allow one to test a hypothesis about a parameter under different data likelihoods and are based on the methodology developed and promoted by Zoltan Dienes (e.g., Dienes 2014; Dienes & Mclatchie, 2018).
```
Comment (AlexanderLyNL) [inline inst/help/summarystatsgeneralbayesiantests.md:4]: "data likelihoods" is by the way odd. A likelihood function is the pdf/pmf fixed at the data treated as a function of the parameters. What about:

The general Bayesian tests allow one to test a hypothesis about a parameter for various distributions and are based on the methodology developed and promoted by Zoltan Dienes (e.g., Dienes 2014; Dienes & Mclatchie, 2018).

Comment (FBartos) [reply inst/help/summarystatsgeneralbayesiantests.md:4]: sure. Any merging issues? (I didn't see them on my end before either)

Comment (EJWagenmakers) [reply inst/help/summarystatsgeneralbayesiantests.md]: "for various likelihood functions"?

Comment (FBartos) [reply inst/help/summarystatsgeneralbayesiantests.md]: updated

**Review by AlexanderLyNL — APPROVED** [review]

**Pull request merged**